app/pages/nonsipp/shares/SharesCompleted.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.shares
+
+import config.Refined.Max5000
+import models.SchemeId.Srn
+import pages.{IndexedQuestionPage, QuestionPage}
+import play.api.libs.json.JsPath
+import utils.RefinedUtils.RefinedIntOps
+import viewmodels.models.SectionCompleted
+
+case class SharesCompleted(srn: Srn, index: Max5000) extends QuestionPage[SectionCompleted] {
+
+  override def path: JsPath =
+    Paths.shareTransactions \ toString \ index.arrayIndex.toString
```

Comment:
Just a question really as a matter of interest. Other 'completed' pages, e.g. `EmployerContributionsCompleted` are having a path set up to be on root, e.g. `override def path: JsPath = JsPath \ ......`
This is now relative to shareTransactions. Any specific reasons to have it there?

Reply:
Just easier to see when you want to look at the raw data (completed shares data next to other shares data) - must have been an oversight with the others but as it doesn't get sent to ETMP it won't be of any consequence